modules/user/user.service.js

```diff
@@ -151,11 +164,34 @@ class UserService {
       ],
     });
     const savedUser = await user.save();
+    const token = await generateToken(savedUser._id, savedUser.email);
+    const message = {
+      from: `horondi.devproject@gmail.com`,
+      to: savedUser.email,
+      subject: 'Confirm Email',
+      html: confirmationMessage(firstName, token),
+    };
+    await sendEmail(message);
     return savedUser;
   }
 
   deleteUser(id) {
     return User.findByIdAndDelete(id);
   }
+
+  async confirmUser(token, language) {
+    try {
+      const decoded = jwt.verify(token, process.env.SECRET);
+      const user = await User.findOne({ email: decoded.email });
+      if (!user) {
+        return new Error(TOKEN_NOT_VALID[language].value);
```

Comment:
Revert this change, please, we are going to send back unlocalized error messages and localize them only on backend

Reply:
removed